src/shared/components/FileSelect/FileSelect.tsx

```diff
@@ -67,49 +91,83 @@ export const FileSelect: React.FC<FileSelectProps> = ({
     noKeyboard: true,
   })
 
+  const [openErrorDialog, closeErrorDialog] = useDialog()
+
+  useEffect(() => {
+    if (!error) {
+      return
+    }
+    openErrorDialog({
+      title: 'Unsupported file type selected',
+      description: error,
```

Comment:
Maybe let's use the step description here? The current dialog description is mostly what's in the title already

<img width="480" alt="Screenshot 2021-09-21 at 11 55 21" src="https://user-images.githubusercontent.com/12646744/134151000-c1682616-5559-410f-b4a6-af1d988abd90.png">